odoo/addons/test_performance/tests/test_performance.py

```diff
@@ -610,3 +613,47 @@ def test_relational_mapped(self):
         with self.assertQueryCount(3):
             for rec in recs:
                 rec.line_ids.mapped('value')
+
+
+@tagged('increment_perf')
+class TestIncrementFieldsSkipLock(TransactionCase):
+    def setUp(self):
+        super().setUp()
+
+        self.record = self.env['test_performance.mozzarella'].create([{
+            'value': 1,
+            'value_plus_one': 2,
+        }])
+
+    def test_increment_fields_skiplock_one_field(self):
+        with self.assertQueryCount(1):
+            updated = sql.increment_fields_skiplock(self.record, 'value')
+            _logger.info('increment_fields_skiplock did %supdate the field', '' if updated else 'not ')
+
+        # increment_fields_skiplock does not invalidate the cache
+        self.record.invalidate_recordset()
+
+        with self.assertQueryCount(1):  # Read
+            if updated:
```

Comment:
Would explain in docstring that we take into account that the "increment_fields_skiplock" can "fail" and return false, in which case it does not increment anything.
Same for the other test.

(When first reading this made me a bit confused ^^)